casa_to_spectralcube/spectralcube_casa_performance.ipynb

```diff
@@ -0,0 +1,349 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.9-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python_defaultSpec_1599753980546",
+   "display_name": "Python 3.6.9 64-bit ('casa_py36': conda)"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Performance comparisons of spectral-cube and CASA #\n",
+    "\n",
+    "## Authors ##\n",
+    "[Eric Koch](https://github.com/e-koch), [Adam Ginsburg](https://github.com/keflavich), [Tom Robitaille](https://github.com/astrofrog)\n",
+    "\n",
+    "## Summary ##\n",
+    "\n",
+    "This tutorial compares the performance of common operations in spectral-cube and CASA. This provides an idea for which operaitons in the respective packages are well optimized. The goal is also to explore how to produce equivalent comparisons since the package use different methods to parallelize operations.\n",
```

Comment:
typo: operations

Reply:
Also I'm not sure about calling this a tutorial, maybe just referring to it as a 'notebook' would make more sense? (and linking to the other notebook in this PR for the average user)